### PR TITLE
Postman instructions update

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Updated eye icon image, some tweaking of the page, updating directions
- **Is this PR related to a Monolith release?**  Nope

### Resolves:

* [OKTA-757253](https://oktainc.atlassian.net/browse/OKTA-757253)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rest/index.md](https://github.com/okta/okta-developer-docs/blob/0904f541c5d8334d822a1580052a72eec9ea7d8d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frest%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9715aaf3-9c13-4b49-aa9f-9a3068897fda) | 80  | 80  | 81  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.1fda80eb-2219-445a-93cb-598d0fd8059d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/rest/index.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ You can then [send test requests](#send-a-request) to your Okta org and verify t
...
+You need a free Okta Developer Edition org to get started. Don't have one? [Create an org for free](https://developer.okta.com/signup). The Okta Developer Edition org is assigned a base URL such as `dev-1234567.okta.com`. This is your unique subdomain in Okta.
 
 > **Note:** Use this unique subdomain whenever you see the `{yourOktaDomain}` variable in this document.
 
 ## Set up your Postman environment
 
 1. [Install the Postman app](https://www.getpostman.com/apps).
 1. Start Postman if it's not open already.
-1. In the upper-left corner, click **Import**.
+1. In the upper-left corner, click the hamburger menu > **File** > **Import** if you're on Windows or **Import** if you're on MacOS.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rest/index.md:30]:
```suggestion
1. In the upper-left corner, click the hamburger menu > **File** > **Import** if you're on Windows. Click **Import** if you're on MacOS.
```

File: packages/@okta/vuepress-site/docs/reference/rest/index.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ You need a free Okta developer-edition org to get started. Don't have one? [Crea
 
 ## Import a collection
 
-Go to the Okta [Postman collections](https://developer.okta.com/docs/reference/postman-collections/) reference to import the collection you want to test.
+Go to the Okta [Postman Collections](https://developer.okta.com/docs/reference/postman-collections/) reference to import the collection you want to test.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rest/index.md:63]:
```suggestion
Go to the Okta [Postman Collections](https://developer.okta.com/docs/reference/postman-collections/) reference to import the collection that you want to test.
```

File: packages/@okta/vuepress-site/docs/reference/rest/index.md [hunk 3]
```diff
@@ -343,7 +341,7 @@ Your imported collections contain URLs and JSON request bodies that have sample
 
   </div>
 
-2. Copy the `id` of the resource, in this example the `id` for Tony Stark, for use in your next request. You can add the `id` value in your Postman environment and use the corresponding variable in your request URL or body.
+1. Copy the `id` of the resource, in this example the `id` for Tony Stark, for use in your next request. You can add the `id` value in your Postman environment and use the corresponding variable in your request URL or body.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rest/index.md:344]:
```suggestion
1. Copy the `id` of the resource for use in your next request. In this example, use the `id` for Tony Stark. You can add the `id` value in your Postman environment and use the corresponding variable in your request URL or body.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rest/index.md](https://github.com/okta/okta-developer-docs/blob/56277a99ddd8d0544ddac47584415a5554521606/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frest%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/31aa193d-ef90-4a55-a9cf-fc52676c14cf) | 80  | 80  | 81  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.1d4c1173-3491-4e04-9343-9715fb2c9599?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rest/index.md](https://github.com/okta/okta-developer-docs/blob/29920304928a1b2dacad32ecf6cb10ec97990dc6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frest%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a3c67586-05be-4bd0-a652-b6249d3e359b) | 80  | 80  | 81  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8da6f7fe-51fe-4a0b-89e8-4888c46451f3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rest/index.md](https://github.com/okta/okta-developer-docs/blob/da23c13a48a347bf25daff8a102b7ffb4572667e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frest%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/511b7957-9631-4c8e-8df0-26f8e173d317) | 80  | 80  | 81  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.cb49ab0d-c042-4832-961f-6f6328b4d665?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Review by danielmaharry-okta — APPROVED** [review]
Looks good to me.

**Pull request merged**